pypeman/contrib/http.py

```diff
@@ -92,14 +92,15 @@ def __init__(self, *args, endpoint=None, method='*', url='/', encoding=None, **k
 class HttpRequest(nodes.BaseNode):
     """ Http request node """
```

Comment:
Can you enhance documentation to explain node params ?

Reply:
better ?